server/service/testing_utils.go

```diff
@@ -575,6 +575,9 @@ func mdmAppleConfigurationRequiredEndpoints() [][2]string {
 		{"GET", "/api/latest/fleet/mdm/apple"},
 		{"GET", apple_mdm.EnrollPath + "?token=test"},
 		{"GET", apple_mdm.InstallerPath + "?token=test"},
+		{"GET", "/api/latest/fleet/mdm/apple/setup/eula/token"},
```

Comment:
This one is unauthenticated, I don't think it should be here? On the other hand, `POST /api/latest/fleet/mdm/apple/setup/eula` should be here.

Reply:
> This one is unauthenticated, I don't think it should be here?

but still requires MDM to be configured! I would expect it to fail if it's not configured

> POST /api/latest/fleet/mdm/apple/setup/eula should be here.

you're right, the problem is that the back-end tries to read the multi-part form that this endpoint is expecting before it gets to the "MDM configured" check.

I'll see how we can workaround that.

Reply:
Ah gotcha, I was going by the comment in `handler.go` that says:

```
	// NOTE: remember to update `service.mdmAppleConfigurationRequiredEndpoints`
	// when you add an endpoint that's behind the mdmConfiguredMiddleware.
```

Looks like it's a bit wider than just those endpoints (I believe that's because those mdm-configured endpoints have the MDM check done automatically, but others can have it done explicitly). Maybe we should rephrase that note a bit.

> the problem is that the back-end tries to read the multi-part form that this endpoint is expecting before it gets to the "MDM configured" check.

Ah yeah, well worse case I think you should add it to the list but comment it out with a note mentioning this, so that in the future we don't think it is missing.